schema/proto3/gateway.proto

```diff
@@ -19,15 +19,18 @@ syntax = "proto3";
 package alcor.schema;
 
 option java_package = "com.futurewei.alcor.schema";
-option java_outer_classname = "AuxiliaryGateway";
+option java_outer_classname = "Gateway";
 
-enum AuxGatewayType {
+enum GatewayType {
```

Comment:
I was calling it "AuxiliaryGateway" to no confused with general networking "Gateway" for cross subnet communication. I guess I will accept we calling it as "Gateway" since most public cloud provider are calling it this way. :)